.gometalinter.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Deadline": "5m",
+    "Enable": [
+        "errcheck",
+        "gofmt",
+        "interfacer",
+        "structcheck",
+        "vet"
+    ],
+    "EnableGC": true,
+    "Linters": {
+        "errcheck": {
+            "Command": "errcheck -abspath {not_tests=-ignoretests} -ignore github.com/hashicorp/terraform/helper/schema:ForceNew|Set -ignore io:Close"
```

Comment:
May be worth splitting errcheck ignores to a separate file if the list may get longer.

Reply:
Agreed. 👍  Let me know if you want that done now (the extra file seemed like too much overhead for this currently small list).